workers/fetcher.go

```diff
@@ -15,9 +14,7 @@ const base = "https://api.propublica.org/congress/v1/"
 type Chamber string
 
 const (
-	//House - House of representatives
 	House Chamber = "house"
-	//Senate - Senate
```

Comment:
![image](https://user-images.githubusercontent.com/22008758/28086342-4dc6a640-669c-11e7-9bea-b7f97a04341e.png)
@greenac My editor was giving the above warning so I added that comment, Its just a warning if its not required its okay for me